tests/nested/manual/hybrid-fde-recovery-keys/task.yaml

```diff
@@ -94,6 +94,14 @@ prepare: |
 
   gendeveloper1 sign-model < classic.json > classic.model
 
+  # 25.10 gadget require much larger disks
+  # https://github.com/canonical/pc-gadget/commit/814d80acbcc0de4ac799b3bcb67f7ec625d6c53a
+  if [ "${MODEL_VERSION}" == "25.10" ]; then
```

Comment:
are we expecting 26.04 to be different?

Reply:
we don't know yet, I don't see a 26.04 branch only core26 https://github.com/canonical/pc-gadget/blob/26/gadget/gadget-amd64.yaml